netlogo-gui/src/main/prim/etc/_extractrgb.java

```diff
@@ -3,19 +3,38 @@
 package org.nlogo.prim.etc;
 
 import org.nlogo.api.LogoException;
+import org.nlogo.api.AgentException;
+import org.nlogo.api.LogoListBuilder;
 import org.nlogo.core.LogoList;
 import org.nlogo.core.Syntax;
+import org.nlogo.core.SyntaxJ;
```

Comment:
I don't see this `SyntaxJ` import used anywhere.  Can it be removed?  Actually, I think `Syntax` above it could be, too, if you wanted.